src/main/java/com/shippo/model/LabelFileType.java

```diff
@@ -0,0 +1,5 @@
+package com.shippo.model;
+
+public enum LabelFileType {
+	PNG, PDF, PDF_4X6, ZPLII
```

Comment:
This is great catch man. How did you find out? I've opened [PR](https://bitbucket.org/shippotle/shippo/pull-requests/3114) to fix in docs.